ApiBundle/Tests/Transformer/ContentAttributeTransformerTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\Tests\Transformer;
+
+use OpenOrchestra\ApiBundle\Transformer\ContentAttributeTransformer;
+use Phake;
+
+/**
+ * Class ContentAttributeTransformerTest
+ */
+class ContentAttributeTransformerTest extends \PHPUnit_Framework_TestCase
+{
+    protected $contentAttributeString;
+    protected $contentAttribute;
+    protected $transformer;
+
+    /**
+     * set Up
+     */
+    public function setUp()
+    {
+        $this->contentAttribute = Phake::mock('OpenOrchestra\ModelInterface\Model\ContentAttributeInterface');
```

Comment:
as the contentAttribute is only used in one test, I would rather declare the mock in the test method